File: src/test/samples/validSamples/validSample.pas
//#region FirstRegion
program HelloWorld;
begin
  writeln('Hello, World!');
end.
// #endregion

{  #region    Second Region  }
procedure MyProcedure;
begin
  (*    #region    InnerRegion    *)
  writeln('Inside region');
  (*  #endregion   ends InnerRegion  *)

  {#region}
  writeln('Another unnamed region');
  {#endregion}
end;
{   #endregion   }

File: src/test/samples/validSamples/validSample.vb
#Region "FirstRegion"
Dim x As Integer = 42
#End Region

#Region "Second Region"
Public Class MyClass
    #Region    "InnerRegion"    
    Public Sub MyMethod()
    End Sub
    #End Region        ends InnerRegion     

    #Region
    Public Sub MyMethod2()
    End Sub
    #End Region
End Class
#End Region
